File: gallery/src/main/java/net/faithgen/gallery/viewholders/SliderHolder.java
package net.faithgen.gallery.viewholders;

import android.view.View;
import android.widget.ImageView;

import com.google.android.material.floatingactionbutton.FloatingActionButton;
import com.smarteist.autoimageslider.SliderViewAdapter;

import net.faithgen.gallery.R;

public class SliderHolder extends SliderViewAdapter.ViewHolder {
    private ImageView imageView;
    private FloatingActionButton floatingActionButton;
    public SliderHolder(View itemView) {
        super(itemView);
        imageView = itemView.findViewById(R.id.image_view);
        floatingActionButton = itemView.findViewById(R.id.comment_image);
    }

    public ImageView getImageView() {
        return imageView;
    }

    public FloatingActionButton getFloatingActionButton() {
        return floatingActionButton;
    }
}

File: gallery/src/main/java/net/faithgen/gallery/viewholders/SliderHolder.kt
package net.faithgen.gallery.viewholders

import android.view.View
import android.widget.ImageView
import com.google.android.material.floatingactionbutton.FloatingActionButton
import com.smarteist.autoimageslider.SliderViewAdapter
import net.faithgen.gallery.R

/**
 * Holder for the SliderAdapter
 */
final class SliderHolder(val itemView: View) : SliderViewAdapter.ViewHolder(itemView) {
    val imageView: ImageView by lazy { itemView.findViewById<ImageView>(R.id.image_view) }
    val floatingActionButton: FloatingActionButton by lazy { itemView.findViewById<FloatingActionButton>(R.id.image_view) }
}